sinks/hawkular/driver.go

```diff
@@ -153,7 +182,14 @@ func (self *hawkularSink) groupName(p *sink_api.Point) string {
 }
 
 func (self *hawkularSink) idName(p *sink_api.Point) string {
-	n := []string{p.Labels[sink_api.LabelContainerName.Key], p.Labels[sink_api.LabelPodId.Key], p.Name}
+	n := make([]string, 0, 3)
+	if p.Labels[sink_api.LabelPodId.Key] != "" {
+		n = append(n, p.Labels[sink_api.LabelContainerName.Key], p.Labels[sink_api.LabelPodId.Key], p.Name)
+	} else {
+		n = append(n, p.Labels[sink_api.LabelContainerName.Key], p.Labels[sink_api.LabelHostID.Key], p.Name)
+	}
```

Comment:
You could split up the append in three parts since two are exactly the same, as in:

```
n = append(n, p.Labels[sink_api.LabelContainerName.Key])
if p.Labels[sink_api.LabelPodId.Key] != "" {
    n = append(n, p.Labels[sink_api.LabelPodId.Key])
} else {
    n = append(n, p.Labels[sink_api.LabelHostID.Key])
}
n = append(n, p.Name)
```